js/bars.js

```diff
@@ -1128,6 +1176,22 @@ historian.Bars.prototype.tooltipText_ = function(
   }
 
   formattedLines.push(cluster.clusteredCount + ' occurences');
+  if (series.source == historian.historianV2Logs.Sources.SENSORSERVICE_DUMP) {
+    var sensor = getSensorByNumber(cluster.sensorNum);
+    // Show the sampling rate information in the floating window.
+    var level = 'Low';
+    if (sensor.RequestMode != 2) {
```

Comment:
sampling rate would would be low for request mode 2. Does not match lines 820-830.

Would it be better to create a helper function for this as well?